src/modules/commander/Arming/ArmStateMachine/ArmStateMachine.cpp

```diff
@@ -154,21 +154,20 @@ transition_result_t ArmStateMachine::arming_state_transition(vehicle_status_s &s
 
 		// Finish up the state transition
 		if (valid_transition) {
-			bool was_armed = armed.armed;
-			armed.armed = (new_arming_state == vehicle_status_s::ARMING_STATE_ARMED);
-			armed.ready_to_arm = (new_arming_state == vehicle_status_s::ARMING_STATE_ARMED)
```

Comment:
That `ready_to_arm` boolean in `actuator_armed` isn't set anywhere else. Is that intentional? Should we remove it completely then? (it controls the LED blinking speed and is somehow used in IO)

Reply:
It's set here: https://github.com/PX4/PX4-Autopilot/blob/33e45dad4d6fd7a6c9fbd1f5589b5557026ae958/src/modules/commander/Commander.cpp#L3053
I'm halfway through filling the `actuator_armed` message in one single place and then later removing the `ready_to_arm` flag. Such an indication could be useful but not the way it's implemented right now.

Reply:
After the rebase it's this line (there was a conflict in the order of publishing):
https://github.com/PX4/PX4-Autopilot/blob/58a455c6a54ddca98e6bf3dd080edb4ce80c1be1/src/modules/commander/Commander.cpp#L3092

Reply:
My bad, I missed it.